src/ResourceManager/Compute/Commands.Compute/Generated/ContainerService/ContainerServiceListMethod.cs

```diff
@@ -35,25 +35,13 @@ public partial class InvokeAzureComputeMethodCmdlet : ComputeAutomationBaseCmdle
         protected object CreateContainerServiceListDynamicParameters()
         {
             dynamicParameters = new RuntimeDefinedParameterDictionary();
-            var pResourceGroupName = new RuntimeDefinedParameter();
-            pResourceGroupName.Name = "ResourceGroupName";
```

Comment:
This file does not produce a cmdlets.  The reason for the removal is the method name in Swagger spec is changed from List to ListByResourceGroup and from ListAll to List.  (This naming was enforced by OneSDK.)  So ListMethod is now ListAll in the previous.

Anyway, this does not affect anything in Powershell because everything is handled in Get- cmdlet properly.  So removal of this file does not have any affect.